Source/WebCore/css/calc/CSSCalcValue.cpp

```diff
@@ -238,16 +238,14 @@ static RefPtr<CSSCalcExpressionNode> createCSS(const CalcExpressionNode& node, c
         }
         case CalcOperator::Mod:
         case CalcOperator::Rem:
-        case CalcOperator::Round: {
-            auto children = createCSS(operationChildren, style);
-            if (children.size() != 2)
```

Comment:
This is technically changing behavior, but I'm curious to see if this really breaks anything or if it's the right path forward.

Reply:
EWS seems fine with this, maybe domain experts can chime in if there are any concerns!